src/route-sync/application/application_test.go

```diff
@@ -24,9 +25,8 @@ var _ = Describe("Application", func() {
 		sourceFake *routefakes.FakeSource
 		routerFake *routefakes.FakeRouter
 		cfg        = &config.Config{
-			RawNatsServers:            "[{\"Host\": \"host\",\"User\": \"user\", \"Password\": \"password\"}]",
 			NatsServers:               []cfConfig.MessageBusServer{{Host: "host", User: "user", Password: "password"}},
-			RoutingApiUrl:             "https://api.cf.example.org",
+			RoutingAPIURL:             "https://api.cf.example.org",
```

Comment:
Any objection to removing all of the upper-casing here? Any rule on when/when not to case makes this struct pretty inconsistent.